azure/hub/outputs.tf

```
@@ -0,0 +1,49 @@
+output "network" {
+  description = "Output from the network module."
+  value       = module.network
+}
+
+output "id" {
+  description = "Output from the network module."
+  value       = module.network.id
+}
+
+output "firewall" {
+  description = "Output from the firewall."
+  value       = one(module.firewall)
```

Suggestion in tandem with `local.firewall` change
```suggestion
  value       = local.firewall
```